roles/open_zaak_k8s/templates/tls.yml.j2

```
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Secret
+data:
+  tls.crt: "{{lookup('file', '{{ openzaak_tls_secrets.crt }}')|b64encode }}"
+  tls.key: "{{lookup('file', '{{ openzaak_tls_secrets.key }}')|b64encode }}"
```

This should just be variables, and those variables themselves can then be defined as `{{ lookup('file', ...) }}` again, or any other way (Ansible vault encrypted, for example).

```suggestion
  tls.crt: "{{ openzaak_tls_secrets.crt | b64encode }}"
  tls.key: "{{ openzaak_tls_secrets.key | b64encode }}"
```

---

Done